lib/gulp-index-file-stream.js

```diff
@@ -22,14 +22,13 @@ IndexFileStream.prototype._read = function() {
   if (this._urls.length == 0)
     return this.push(null);
   var url = this._urls.pop();
-  var indexFile = posixPath.join(
+  var indexFile = path.join(
     this._baseDir,
-    posixPath.join.apply(posixPath, url.split('/').slice(1, -1)),
+    path.join.apply(path, url.split('/').slice(1, -1)),
     'index.html'
   );
 
   this._indexStatic.generate(url, {
```

Comment:
I don't know if we need second parameter here anymore? What do you think @toolness ?